Document the `res.redirect()` helper function for `@vercel/node` (#2090)
vercel/docs: components/references-mdx/runtimes/official-runtimes/official-runtimes.mdx

| `res.status(code)`                                                                         | A function to set the status code sent with the response where `code` must be a valid [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). Returns `res` for chaining.                                                                                                     | Response |
| `res.send(body)`                                                                           | A function to set the content of the response where `body` can be a `string`, an `object` or a `Buffer`.                                                                                                                                                                                           | Response |
| `res.json(obj)`                                                                            | A function to send a JSON response where `obj` is the JSON object to send.                                                                                                                                                                                                                         | Response |

> | `res.redirect(statusCodeOrlUrl, [url])`                                                    | A function to set a redirect, redirects to the URL derived from the specified path, with specified status, a positive integer that corresponds to an [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). If not specified, status defaults to “307“ Temporary Redirect ”. | Response |

Reviewer: It would be better to list the two syntaxes separately I think. Maybe this would work:

```suggestion
| `res.redirect(url)`<br/>`res.redirect(statusCode, url)`                                                    | A function to set a redirect, redirects to the URL derived from the specified path, with specified status, a positive integer that corresponds to an [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). If not specified, status defaults to "307 Temporary Redirect". | Response |
```

Reviewer 2: @TooTallNate What do you think about `status` instead of `statusCode` to make it shorter?

Reviewer: Either works for me. But even the shorter `status` cause a line break due to spacing 😒

Author: Try to solve it this way. I have the same opinion as @TooTallNate , either of the 2 works for me too.
<img width="770" alt="Screen Shot 2020-08-20 at 10 15 42" src="https://user-images.githubusercontent.com/16585386/90791586-ff255b00-e2ce-11ea-9c68-5a181a586454.png">

| `res.status(code)`                                                                         | A function to set the status code sent with the response where `code` must be a valid [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). Returns `res` for chaining.                                                                                                    | Response |
| `res.send(body)`                                                                           | A function to set the content of the response where `body` can be a `string`, an `object` or a `Buffer`.                                                                                                                                                                                          | Response |
| `res.json(obj)`                                                                            | A function to send a JSON response where `obj` is the JSON object to send.                                                                                                                                                                                                                        | Response |

> | `res.redirect(url)`                                                                        | A function to set a redirect, redirects to the URL derived from the specified path, with specified status, a positive integer that corresponds to an [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). If not specified, status defaults to “307 Temporary Redirect ”. | Response |

Reviewer 2: ```suggestion
| `res.redirect(url)`                                                                        | A function to redirect to the URL derived from the specified path with status code “307 Temporary Redirect ”. | Response |
```

> | `res.redirect(statusCode, url)`                                                            | A function to set a redirect, redirects to the URL derived from the specified path, with specified status, a positive integer that corresponds to an [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). If not specified, status defaults to “307 Temporary Redirect ”. | Response |

Reviewer 2: ```suggestion
| `res.redirect(statusCode, url)`                                                            | A function to redirect to the URL derived from the specified path, with specified [HTTP status code](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes). | Response |
```

Author: Amazing work. You shortened much of the previous definition by making it shorter and more declarative 🎉 👍

Author: the new definitions now look much better.
